### PR TITLE
(SIMP-4398) Fix .gitlab-ci.yml syntax error

### PR DESCRIPTION
**Pull request by lnemsick-simp**
SIMP-4398 #close

### CODE REVIEW

File: .fixtures.yml [hunk 1]
```diff
@@ -4,7 +4,9 @@ fixtures:
     compliance_markup: https://github.com/simp/pupmod-simp-compliance_markup
     augeasproviders_core: https://github.com/simp/augeasproviders_core
     augeasproviders_grub: https://github.com/simp/augeasproviders_grub
-    concat: https://github.com/simp/puppetlabs-concat
+    concat:
+      repo: https://github.com/simp/puppetlabs-concat
+      ref: 3.0.0
```
Comment (jeefberkey) [inline .fixtures.yml]: Pin to 3.0.0?

Comment (trevor-vaughan) [reply .fixtures.yml]: @jeefberkey If you think that should be changed, go ahead and update it and we'll revisit this after the tests run. `4.1.1` is the latest release.

@lnemsick-simp Thoughts on the pinning comment?

Comment (lnemsick-simp) [reply .fixtures.yml]: @trevor-vaughan We have many modules for which the upperbound is pinned to < 4.0.0.  So, I wanted to handle those metadata.json updates, first, and then up the pinning.

Comment (jeefberkey) [reply .fixtures.yml]: I created a ticket upstream to track this issue: https://tickets.puppetlabs.com/browse/MODULES-6625

Comment (trevor-vaughan) [reply .fixtures.yml]: 👍

**Pull request merged**